docs/tutorial/getting-started.md

```diff
@@ -0,0 +1,43 @@
+---
+id: getting-started
+title: Getting started with Tartiflette
+sidebar_label: 1. Getting started
+---
+
+Welcome, either you are a GraphQL newbie or an advanced user, Tartiflette will help you to build a powerful GraphQL API.
+
+This tutorial will explain to you the main features of Tartiflette and some of its advanced features _(dynamic introspection, SDL Directives ...)_ 
```

Comment:
will explain ~to you~ the main features of Tartiflette as well as some more advanced features

Reply:
👍 

Change by 

```
will explain to you the main features of Tartiflette as well as some more advanced ones
```